AGENTS.md

```
@@ -1 +1,147 @@
-See AGENTS.md for all agent guidelines.
+# Multi-Agent Coordination
+
+This document coordinates work between AI agents working on this repository.
+
+## Active Agents
+
+| Agent | IDE | Responsibilities | Status |
+|-------|-----|------------------|--------|
+<<<<<<< HEAD
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)